docs/reference/release-notes/7.7.asciidoc

```diff
@@ -1,22 +1,612 @@
+* Validate role templates before saving role mapping {pull}52636[#52636] (issue: {issue}48773[#48773])
+* Add support for secondary authentication {pull}52093[#52093]
+* Expose API key name to the ingest pipeline {pull}51305[#51305] (issues: {issue}46847[#46847], {issue}49106[#49106])
+* Disallow Password Change when authenticated by Token {pull}49694[#49694] (issue: {issue}48752[#48752])
+
+Authorization::
+* Allow kibana_system to create and invalidate API keys on behalf of other users {pull}53824[#53824] (issue: {issue}48716[#48716])
+* Add "grant_api_key" cluster privilege {pull}53527[#53527] (issues: {issue}48716[#48716], {issue}52886[#52886])
+* Giving kibana user privileges to create custom link index {pull}53221[#53221] (issue: {issue}59305[#59305])
+* [APM] Allow kibana to collect APM telemetry in background task {pull}52917[#52917] (issue: {issue}50757[#50757])
+* Add the new 'maintenance' privilege containing 4 actions (#29998) {pull}50643[#50643]
+
+Cluster Coordination::
+* Describe STALE_STATE_CONFIG in ClusterFormationFH {pull}53878[#53878] (issue: {issue}53734[#53734])
+
+Distributed::
+* Introduce formal role for remote cluster client {pull}53924[#53924]
+* Shortcut query phase using the results of other shards {pull}51852[#51852] (issues: {issue}49601[#49601], {issue}51708[#51708])
+* Flush instead of synced-flush inactive shards {pull}49126[#49126] (issues: {issue}31965[#31965], {issue}48430[#48430])
+
+Engine::
+* Restore off-heap loading for term dictionary in ReadOnlyEngine {pull}53713[#53713] (issues: {issue}43158[#43158], {issue}51247[#51247])
+* Separate translog from index deletion conditions {pull}52556[#52556]
+* Always rewrite search shard request outside of the search thread pool {pull}51708[#51708] (issue: {issue}49601[#49601])
+* Move the terms index of `_id` off-heap. {pull}52405[#52405] (issue: {issue}42838[#42838])
+* Cache completion stats between refreshes {pull}51991[#51991] (issue: {issue}51915[#51915])
+* Use local checkpoint to calculate min translog gen for recovery {pull}51905[#51905] (issue: {issue}49970[#49970])
+
+Features/CAT APIs::
+* /_cat/shards support path stats {pull}53461[#53461]
+* Allow _cat indices & aliases to use indices options {pull}53248[#53248] (issue: {issue}52304[#52304])
+
+Features/Features::
+* Enable deprecation checks for removed settings {pull}53317[#53317]
+
+Features/ILM+SLM::
+* Hide ILM & SLM history aliases {pull}53564[#53564]
+* Avoid race condition in ILMHistorySotre {pull}53039[#53039] (issues: {issue}50353[#50353], {issue}52853[#52853])
+* Make FreezeStep retryable {pull}52540[#52540]
+* Make DeleteStep retryable {pull}52494[#52494]
+* Allow forcemerge in the hot phase for ILM policies {pull}52073[#52073] (issue: {issue}43165[#43165])
+* Stop policy on last PhaseCompleteStep instead of TerminalPolic… {pull}51631[#51631] (issue: {issue}48431[#48431])
```

Comment:
PR label was truncated.
```suggestion
* Stop policy on last PhaseCompleteStep instead of TerminalPolicyStep {pull}51631[#51631] (issue: {issue}48431[#48431])
```